Cargo.toml

```diff
@@ -260,8 +260,8 @@ unwrap_used = "deny"
 use_debug = "deny"
 
 [profile.release]
-codegen-units = 1
-lto = "thin"      # attempts to perform optimizations across all crates within the dependency graph
+#codegen-units = 1
```

Comment:
don't merge this

Reply:
thanks for catching this, it's very annoying that the fuzzer only ever compiles release :(